### PR TITLE
Adding ellipsis on break points

### PR DESCRIPTION
**Pull request by discodavey**

### CODE REVIEW

**Comment by discodavey** [comment]
Got to think about something so it works properly. Not ready for review

**Comment by discodavey** [comment]
Chris has approved the front end visual. Just need the code to be reviewed.

**Review by NuclearRedeye — APPROVED** [review]
Tested locally and it looks OK to me

File: source/_patterns/00-atoms/references/reference~data.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "title": "NKX2-5 mutations causative for congenital heart disease retain functionality and are directed to hundreds of targets",
-  "origin": "NCBI Gene Expression Omnibus. Series accession number <a href=\"https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE44902\">GSE44902</a>",
+  "origin": "NCBI Gene Expression Omnibus. Series accession number (<a href=\"https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE44902\">GSE44902</a>)",
```
Comment (nlisgo) [inline source/_patterns/00-atoms/references/reference~data.json:3]: Why is this change included?

File: source/_patterns/04-pages/article--research.json [hunk 2]
```diff
@@ -1101,6 +1101,36 @@
       },
       "hasAbstracts": false
     },
+    {
+      "bibId": "bib61",
```
Comment (nlisgo) [inline source/_patterns/04-pages/article--research.json:1105]: There are examples of other references in this example page that include a titleLink, what does this add?

File: source/_patterns/00-atoms/references/reference~data.json (hunk 1 above)
Comment (discodavey) [reply source/_patterns/00-atoms/references/reference~data.json:3]: This is due to there are brackets around this item which Chris mentioned

**Review by nlisgo — CHANGES_REQUESTED** [review]
Screenshot 2021-03-16 at 19 20 17

There is an issue with the spacing between references.

See: https://s3.amazonaws.com/ci-pattern-library/1063/patterns/01-molecules-components-reference-list/01-molecules-components-reference-list.html

**Comment by discodavey** [comment]
I had done the atoms but not the molecules and up as we were on first pass.

File: assets/sass/patterns/atoms/reference.scss [hunk 3]
```diff
@@ -123,12 +127,33 @@ a.reference__authors_link:hover {
...
+    }
+  }
+
   .reference__origin {
 
     a {
       @include font-size-and-vertical-height(14);
       border-bottom: none;
       color: $color-primary;
+
```
Comment (nlisgo) [inline assets/sass/patterns/atoms/reference.scss:156]: nitpick: remove whitespace

**Review by nlisgo — APPROVED** [review]

File: assets/sass/patterns/atoms/reference.scss [hunk 4]
```diff
@@ -136,7 +161,7 @@ a.reference__authors_link:hover {
     display: inline;
   }
 
-  .doi + .reference__abstracts > .reference__abstract,
+  .reference__abstracts > .reference__abstract,
```
Comment (nlisgo) [inline assets/sass/patterns/atoms/reference.scss:164]: why is this change needed?

Comment (nlisgo) [reply assets/sass/patterns/atoms/reference.scss:164]: wouldn't this impact `.doi` links?

Comment (discodavey) [reply assets/sass/patterns/atoms/reference.scss:164]: This had been removed before but not sure why it returned.
